apps/wallet-mobile/src/features/Settings/EasyConfirmation/EnableEasyConfirmationScreen.tsx

```diff
@@ -115,27 +119,33 @@ const messages = defineMessages({
   },
 })
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-  },
-  contentContainer: {
-    padding: 16,
-  },
-  heading: {
-    fontSize: 16,
-    lineHeight: 24,
-    paddingBottom: 20,
-  },
-  warning: {
-    color: COLORS.RED,
-    fontSize: 14,
-    lineHeight: 20,
-    paddingBottom: 20,
-  },
-  actions: {
-    paddingBottom: 16,
-    paddingHorizontal: 16,
-  },
-})
+const useStyles = () => {
+  const {theme} = useTheme()
+  const {color} = theme
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      backgroundColor: color.gray.min,
+    },
+    contentContainer: {
+      padding: 16,
+    },
+    heading: {
+      fontSize: 16,
```

Comment:
can it consume any typography?

Reply:
yes!l let me check all the `fontSize` again